.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '21.0.0'
+          java-version: '21.0.2'
```

Comment:
The last workflow failed with:
Your Java runtime '21+35-jvmci-23.1-b15' with compiler version '23.1.1' is incompatible with polyglot version '23.1.2'.

Reply:
That is expected. thanks